bukkit/src/main/resources/plugin.yml

```diff
@@ -10,6 +10,7 @@ libraries:
   - 'com.mysql:mysql-connector-j:${mysql_driver_version}'
   - 'org.mariadb.jdbc:mariadb-java-client:${mariadb_driver_version}'
   - 'org.xerial:sqlite-jdbc:${sqlite_driver_version}'
+  - 'io.netty:netty-all:${netty_version}'
```

Comment:
we shouldn't need to include this, right? Like pretty sure the Minecraft server bundles this